railties/Rakefile

```diff
@@ -71,6 +71,7 @@ namespace :test do
 
     dirs = (ENV["TEST_DIR"] || ENV["TEST_DIRS"] || "**").split(",")
     test_options = ENV["TESTOPTS"].to_s.split(/[\s]+/)
+    test_options << "--profile" if ENV["CI"]
```

Comment:
This doesn't seem to be working on `railties` tests, which is the one I'm most interested about.